qdrant/manifest.json

```diff
@@ -0,0 +1,60 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bee8e975-219c-4673-83c0-9fa68b973298",
+  "app_id": "qdrant",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "A high-performance vector search engine/database.",
+    "title": "Qdrant",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "A Qdrant Integration Dashboard for Datadog",
+        "image_url": "images/qdrant_dashboard.png"
+      }
```

Comment:
we require 3 images for the media carousel. The images included here will display on your tile

Reply:
[`094a8ee` (#2422)](https://github.com/DataDog/integrations-extras/pull/2422/commits/094a8ee3c78d7d24d9e2a3b371d19c4ba69a9276)